packages/frontend/editor-ui/src/features/shared/banners/components/BannerStack.vue

```diff
@@ -36,25 +38,42 @@ import { computed, onMounted } from 'vue';
 import { getBannerRowHeight } from '@/utils/htmlUtils';
 
 const uiStore = useUIStore();
+const dynamicBannersStore = useDynamicBannersStore();
```

Comment:
Does it make sense to handle all banners (static and dynamic) in this new store?